File: Python3/N_mod_M.py
import sys
import math


def mymod(n: int, m: int) -> int:
    return n - math.floor(n / m) * m

mymod(20, 7)

with open(sys.argv[1], 'r') as test_cases:
    for test in test_cases:
        args = test.split(",")
        n = int(args[0])
        m = int(args[1])
        print(mymod(n, m))

File: Lua/N_mod_m.lua
-- a % b == a - math.floor(a/b)*b
function mymod(n,m)
  return n - math.floor(n/m)*m
end

print(mymod(20,6))
line = "20,6"
local args = {}
line:gsub("[^,]+", function(x) args[#args + 1] = tonumber(x) end)
print(table.concat(args, " "))

for line in io.lines(arg[1]) do
  local args = {}
  line:gsub("[^,]+", function(x) args[#args + 1] = tonumber(x) end)
  print(mymod(args[1], args[2]))
end
